src/dotnet/APIView/APIViewWeb/Pages/Assemblies/_CodeLine.cshtml

```diff
@@ -26,7 +26,14 @@
         <td colspan="2">
             @foreach (var lineDiagnostic in Model.Diagnostics)
             {
-                <p>
+                string diagClass = lineDiagnostic.Level switch
+                {
+                    APIView.CodeDiagnosticLevel.Error => "code-diagnostics-error",
```

Comment:
You can remove this line the `                    _ => "code-diagnostics-error"
` would handle it